tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -21,50 +22,42 @@ internal abstract class LambdaCommon
     private const double ServerlessMaxWaitingFlushTime = 3;
     private const string LogLevelEnvName = "DD_LOG_LEVEL";
 
-    internal static Scope CreatePlaceholderScope(Tracer tracer, string traceId, string samplingPriority)
+    internal static Scope CreatePlaceholderScope(Tracer tracer, WebHeaderCollection headers)
     {
-        Span span;
+        var spanContext = SpanContextPropagator.Instance.Extract(headers, SafeGetValues);
```

Comment:
You don't need the helper if you use `WebHeadersCollection`

```suggestion
        var spanContext = SpanContextPropagator.Instance.Extract(headers);
```